app/views/auctions/show.html.erb

```diff
@@ -39,6 +39,12 @@
         <p class="auction-label"><%= @view_model.auction_start_label %></p>
         <p class="auction-label-info"><%= @view_model.auction_formatted_start_time %></p>
       </div>
+      <% if @view_model.auction_has_delivery_deadline? %>
```

Comment:
Well, the method would return `&nbsp;` if the delivery_deadline is not set, but the label would still be there.